capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
+| Cloud-Init                                 | No                        | No              | No                          | Yes                        | No                                    | Yes                               |
+| Sysprep Windows Templates                  | No                        | No              | No                          | Yes                        | No                                    | N/A                               |
+| Network Manager Relationships              | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Relationship Discovery                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Cloud Network Inventory                    | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Cloud Subnet Inventory                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Network Router Inventory                   | No                        | No              | Yes                         | Yes                        | No                                    | No                                |
+| Security Groups                            | Yes                       | Yes             | Yes                         | Yes                        | No                                    | N/A                               |
+| Floating IP Addresses                      | Yes                       | Yes             | Yes                         | Yes                        | No                                    | N/A                               |
+| Network Ports                              | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Load Balancer Inventory                    | Yes                       | Yes             | Yes                         | No                         | No                                    | No                                |
+| Create/Update/Delete Network               | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Create/Update/Delete Subnet                | No                        | No              | No                          | Yes                        | No                                    | No                                |
```

Comment:
We say support create and delete in https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager/cloud_subnet.rb#L2-L4
is the No because create looks like it isn't implemented (is that a bug?) or because update isn't supported?

Reply:
Good catch @agrare. Create and delete to do work* (and update is not implemented). The create form is still pretty messy - lots of n/a fields, missing PowerVS parameters, and `Subnet CIDR` is required to submit but not used by the API call for our "public" networks. We're thinking it would be better to disable subnet create for Kasparov than to try and document using the form in a confusing way. This [form is moving to DDF anyways](https://github.com/ManageIQ/manageiq-ui-classic/issues/6861).

I'll submit a PR to ManageIQ/manageiq-providers-ibm_cloud shortly to disable it.